lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/compose/layer/SymbolLayer.kt

```diff
@@ -504,17 +520,39 @@ public fun SymbolLayer(
       set(iconTranslate) { layer.setIconTranslate(it) }
       set(iconTranslateAnchor) { layer.setIconTranslateAnchor(it) }
 
+      set(textSizeSp to fontScale) { (textSizeSp, fontScale) ->
+        layer.setTextSize((textSizeSp * const(fontScale)).dp)
+      }
```

Comment:
I'd use `Pair(textSizeSp, fontScale)` for clarity (the `to` metaphor doesn't really work here), or, even better (IMO), use the `listOf` API consistently here when more than one parameter is involved.

(I do wonder why the `set` method doesn't simply have a vararg parameter)

Reply:
I'm gonna use remember instead, and pass a single value in here. 

Simplified reimplementation coming in a bit